contribs/gmf/src/datasource/Manager.js

```diff
@@ -243,6 +243,21 @@ export class DatasourceManager {
     this.ngeoDataSources_.map = map;
   }
 
+  /**
+   * Get a datasource by its name.
+   * @param {string} name The name of the datasource.
+   * @hidden
+   */
+  getDatasource(name) {
```

Comment:
It is secure ? can't we work with ids ? It is possible that we have two datasources with the same name ?